rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -2768,11 +2745,11 @@ private boolean verifyLockDoesNotSurpassLockingCap(BtcTransaction btcTx, Coin to
         }
 
         Coin fedCurrentFunds = getBtcLockedInFederation();
-        Coin lockingCap = this.getLockingCap();
+        Optional<Coin> lockingCap = lockingCapSupport.getLockingCap();
```

Comment:
The only case when this returns empry is when RSKIP134 is not active, is that so?

Because if that's the case we could get rid change the part above 
```if (!activations.isActive(ConsensusRule.RSKIP134)) {
            return true;
        }```

for 

```
if (!lockingCap.isPresent()) return true
```
Would make it cleaner and clearer I believe

Reply:
Done! Removed the `IF` statement related to `activations`, and added a new validation related to the `LockingCap` value to improve Readiness and Maintainability.